### ISSUE TITLE
🆕 Software Suggestion | InviZible Android

### ISSUE THREAD

**Issue by samuraikidXDA** [privacytools/privacytools.io#2204]
## Basic Information

**Name:** InviZible
**Category:** Android Privacy addons
**URL:** https://invizible.net/en/

## Description

Keeps privacy, protects your device from dangerous sites, prevents tracking, gets access to blocked on-line resources

InviZible Pro includes well-known modules such as DNSCrypt, Tor and Purple I2P. They are used to achieve maximum security, privacy and comfortable use of the Internet.

## Why I am making the suggestion

Increase everyone privacy, an important thing this days

## Anything you would like to tell us about the software?

Very good, easy use root and no root options

## My connection with the software

Nop just and Enthustiastic

Cheers

**Comment by freddy-m** [comment]
Duplicate of #1586
